src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,180 @@
+package main.java.duke;
+
+import main.java.duke.task.Task;
+
+/**
+ * Encapsulates the user interface of duke
+ */
+public class Ui {
+
+    /** String to print upon adding a new task to the task list */
+    private final String addTaskMessage = "Got it. I've added this task:";
```

Comment:
Should final variables be all caps?

Reply:
Also should associated constants be prefixed by a common type name? so that when sorted alphabetically they will appear together.